app/views/proposal_types/_form.html.erb

```diff
@@ -28,6 +28,11 @@
             <%= f.text_field :name %>
           </div>
 
+          <div class="mb-3">
+            <%= f.label :year, class: 'form-label' %><br>
+            <%= f.text_field :year, class: 'form-control' %>
+          </div>
+
           <div class="mb-3">
             <%= f.label :Location , class: 'form-label' %><br>
             <%= f.select :location_ids, locations, {}, class: 'form-control', multiple: true %>
```

Comment:
Should say that this text field supports a list of years; as-is it looks like just 1 year.